pytest_watch/command.py

```diff
@@ -11,7 +11,7 @@
   --ignore <dir>        Ignore directory from being watched and during
                         collection (multi-allowed).
   --ext <exts>          Comma-separated list of file extensions that can
-                        trigger a new test run when changed (default: .py).
+                        trigger a new test run when changed [default: .py].
```

Comment:
~Are brackets a common for default values in CLI tool `--help` texts?~ Nevermind. Just noticed docopt uses brackets [in their docs](http://docopt.org/). Thanks for updating that.